fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsAccountReadPlatformServiceImpl.java

```diff
@@ -200,6 +200,30 @@ public SavingsAccountReadPlatformServiceImpl(final PlatformSecurityContext conte
         return this.paginationHelper.fetchPage(this.jdbcTemplate, sqlCountRows, sqlBuilder.toString(), finalObjectArray,
                 this.savingAccountMapper);
     }
+    
+    //Pagination Read service for Active Savings Account Post Interest Job
+    @Override
+    public Page<SavingsAccountData> retrieveActiveSavingsAccount(final Integer maxPazeSize,final Integer pageNumber, final Integer status) {
```

Comment:
The method itself is to retrieve active savings accounts. Do we need to pass status as param?